lib/util/wallet_ext.dart

```diff
@@ -0,0 +1,24 @@
+import 'dart:convert';
+
+import 'package:autonomy_flutter/common/injector.dart';
+import 'package:autonomy_flutter/model/pair.dart';
+import 'package:autonomy_flutter/service/tezos_service.dart';
+import 'package:autonomy_flutter/util/wallet_storage_ext.dart';
+import 'package:flutter/foundation.dart';
+import 'package:libauk_dart/libauk_dart.dart';
+
+extension WalletExt on Pair<WalletStorage, int> {
+  Future<Map<String, dynamic>> get chatAuthBody async {
```

Comment:
it is weird to me to have an async property
i rather have await .chatAuthBody() then await .chatAuthBody